react/Viewer/index.jsx

```diff
@@ -119,7 +119,7 @@ export const toolbarPropsPropType = {
 
 Viewer.propTypes = {
   /** One `io.cozy.files` to display */
-  currentFile: PropTypes.shape(FileDoctype).isRequired,
+  currentFile: PropTypes.shape(FileDoctype),
```

Comment:
You remove the required because of the documentation?

Reply:
Non, un warning était déjà présent et causé par le fait que l'on devrait directement utiliser `FileDoctype.isRequired`
Je viens de le fixer